src/main/java/com/brew/oauth20/server/provider/authorizetype/BaseAuthorizeTypeProvider.java

```diff
@@ -20,6 +20,6 @@ public ValidationResultModel validate(String clientId, String redirectUri) {
         if (clientModel == null)
             return new ValidationResultModel(false, "unauthorized_client");
 
-        return new ClientValidator(clientModel).validate(responseType.getResponseType(), redirectUri);
+        return new ClientValidator(clientModel).validate(responseType.getResponseType(), redirectUri, null);
```

Comment:
It would be better to override the method instead of passing null with a common method.